lib/vet360/person/service.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'vet360/contact_information/transaction_response'
+
+module Vet360
+  module Person
+    class Service < Vet360::Service
+      include Common::Client::Monitoring
+
+      OID = MVI::Responses::IdParser::CORRELATION_ROOT_ID
+      AAID = MVI::Responses::IdParser::VET360_ASSIGNING_AUTHORITY_ID
+
+      configuration Vet360::ContactInformation::Configuration
+
+      def init_vet360_id(icn)
+        with_monitoring do
+          raw_response = perform(:post, encoded_uri_for(icn), empty_body, skip_vet360_id: true)
+
+          Vet360::ContactInformation::PersonResponse.from(raw_response)
+        end
+      rescue StandardError => e
+        handle_error(e)
```

Comment:
What sort of errors can this endpoint throw? Should we gracefully handle the situation where a user may have already been initialized? A stale MVI cache could conceivably result in this service getting called with already initialized users more frequently than we'd expect. I realize this is also probably a discussion more around the future work for this service, not necessarily for this rake task version. Don't let this block you from merging.